index.html

```diff
@@ -1933,6 +1924,19 @@ <h3>Verification Material</h3>
 an example of a public key with a compound key identifier.
             </p>
           </dd>
+          <dt><dfn>publicKeyMultibase</dfn></dt>
+          <dd>
+            <p>
+The <code>publicKeyMultibase</code> property is OPTIONAL. This feature is
+non-normative. If present, the value MUST be a <a
+data-cite="INFRA#string">string</a> representation of a [[?MULTIBASE]] encoded
+public key.
+            </p>
+            <p class="advisement">
+Note that publicKeyMultibase is not a standard and is therefore subject to change. There might be some cases where <code><b>public</b>KeyMultibase</code> can
+be represented although <code><b>private</b>KeyMultibase</code> cannot.
```

Comment:
```suggestion
Note that the [[?MULTIBASE]] specification is not yet a standard and is 
subject to change. There might be some data format use cases where 
<code><b>public</b>KeyMultibase</code> is defined to express keys 
publicly, but <code><b>private</b>KeyMultibase</code> is not in order 
to protect against accidentally leaking secret keys.
```

Reply:
@OR13 if you're good w/ this change, I can apply it (or you can) and we can merge this PR.

Reply:
these changes look good.